ceph.spec.in

```diff
@@ -153,6 +156,15 @@ BuildRequires:  openldap-devel
 BuildRequires:  openssl-devel
 BuildRequires:  redhat-lsb-core
 BuildRequires:	Cython
+%if 0%{?rhel}
+BuildRequires:	python34-devel
+BuildRequires:	python34-setuptools
+BuildRequires:	python34-Cython
+%else
+BuildRequires:	python3-devel
+BuildRequires:	python3-setuptools
+BuildRequires:	python3-Cython
+%endif
 %endif
```

Comment:
The BuildRequires for suse, fedora, and centos are all the same, I gather. In that case, instead of the above modifications to the suse and fedora/rhel sections, which introduce duplication, could you just put the following lines here:

```

# python34-... for rhel, python3-... for all other supported distros
%if 0%{?rhel}
BuildRequires:	python34-devel
BuildRequires:	python34-setuptools
BuildRequires:	python34-Cython
%else
BuildRequires:	python3-devel
BuildRequires:	python3-setuptools
BuildRequires:	python3-Cython
%endif
```